requirements.txt

```diff
@@ -1,2 +1,5 @@
 docker-py
+flake8
 molecule
+yamllint
+ansible-lint
```

Comment:
Can we peg the proper modules and versions, please?

docker==3.5.1
molecule==2.19.0

Reply:
updated with scaffold :)